pkg/kubelet/dockertools/exec.go

```diff
@@ -54,7 +54,8 @@ func (*NsenterExecHandler) ExecInContainer(client DockerInterface, container *do
 	args = append(args, cmd...)
 	command := exec.Command(nsenter, args...)
 	if tty {
-		p, err := kubecontainer.StartPty(command)
+		var p *os.File
+		p, err = kubecontainer.StartPty(command)
```

Comment:
How about declaring a new variable (e.g., `var commandErr error`) before line 56 to reduce the chance of regression again?